engine/bsql_engine/io/io.pyx

```diff
@@ -367,7 +387,9 @@ cpdef runSkipDataCaller(int masterIndex,  tcpMetadata,  table_obj,  vector[int]
     cdef vector[string] currentTableSchemaCppArgKeys
     cdef vector[string] currentTableSchemaCppArgValues
     cdef vector[string] tableNames
-    cdef vector[gdf_column_ptr] columns
+    cdef vector[column_view] columns
```

Comment:
this is a bit confusing to me why are there schemas, new_column and a table_view?

Reply:
To can call the `view()` method in line 456, new_column should be a `Column` type. And you should explicitly said to cython the type.